frontend/integration-tests/views/monitoring.view.ts

```diff
@@ -32,3 +33,66 @@ export const modalConfirmButton = $('#confirm-action');
 export const alertManagerYamlForm = $('.co-alert-manager-yaml__form');
 export const successAlert = $('.pf-m-success');
 export const helpText = $('.co-help-text');
+
+// Configuration Overview
+export const alertRoutingHeader = $$('h2').first();
+export const alertRoutingEditButton = $('.btn-edit-alert-routing');
```

Comment:
Not new, but this class should follow BEM.

Reply:
fixed